packages/nocodb/src/lib/noco/meta/api/dataApis/dataAliasApis.ts

```diff
@@ -187,6 +211,12 @@ router.get(
   ncMetaAclMw(dataFindOne, 'dataFindOne')
 );
 
+router.get(
+  '/api/v1/db/data/:orgs/:projectName/:tableName/groupby',
+  apiMetrics,
+  ncMetaAclMw(dataGroupBy, 'dataGroupBy')
```

Comment:
Please add `dataGroupBy` to `projectAcl.ts` so that the corresponding roles can trigger this api.

Reply:
done